enterprise/backend/test/metabase_enterprise/data_editing/test_util.clj

```diff
@@ -56,15 +66,17 @@
          column-map    (walk/postwalk-replace {'auto-inc-type auto-inc-type} column-map)
          db            (t2/select-one :model/Database (mt/id))
          table-name    (str "temp_table_" (str/replace (random-uuid) "-" "_"))
-         cleanup       #(try (driver/drop-table! driver (mt/id) table-name) (catch Exception _))]
+         cleanup       (fn []
+                         (driver/drop-table! driver (mt/id) table-name)
+                         (t2/delete! :model/Table :name table-name))]
      (try
        (let [table-id (create-test-table! driver db table-name column-map create-table-opts)]
          (reify Closeable
            IDeref
            (deref [_] table-id)
            (close [_] (cleanup))))
        (catch Exception e
-         (cleanup)
+         (try (cleanup) (catch Exception _))
```

Comment:
Why suppress this exception?

Feels bad to just hide it, you could throw a 3rd exception with `{:our-exception e, :inner-exception _}` for full transparency. Not sure if I'm joking.

Reply:
Dude this is copy pasta from the original `with` magic. I actually agree, I just need to add something that wasnt there before (I've just moved where it happens)

Reply:
So yep 👍

Reply:
ef9c5a492ead9b8879c69219577a2cc8bcad5b20 

Got to use .addSurpressed, been a while :)

Reply:
"Dude" 😆 

How have I never learned about that Throwable method before? It's great!